launchpod/src/main/java/com/google/launchpod/data/UserFeed.java

```diff
@@ -0,0 +1,51 @@
+package com.google.launchpod.data;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlRootElement;
+import com.google.appengine.api.datastore.Entity;
+import com.google.launchpod.servlets.FormHandlerServlet;
+
+@JacksonXmlRootElement(localName = "rss")
+public final class UserFeed{
```

Comment:
i think this should be a different object from what we are storing in DataStore. we should discuss pros and cons in the design doc (hint: problem can become more apparent if we write out all the fields we will be asking the user for in the form).

also think about what sort of fields we'll need to store in the datastore that might not be needed for the rss feed